system_tests/Dockerfile

```diff
@@ -4,26 +4,23 @@ FROM google/python
 # TODO: Encrypt/decrypt secrets
 ADD data/ data/
 ENV GOOGLE_APPLICATION_CREDENTIALS /app/data/cloud-python-runtime-qa-credentials.json
+ENV GCLOUD_PROJECT cloud-python-runtime-qa
```

Comment:
Will you file an issue to figure out how to not hardcode these? (I'm thinking just passing them to docker run, but we can address it later).